### PR TITLE
Pathway ready

### LINKED ISSUE #526: List things to change for the Pathway UI
List things to change for the Pathway UI

For instance: 
Categorize the pathways.
Give better instructions

### LINKED ISSUE #527: Design the new UI for Pathway
Design the new UI for Pathway,
provide sketches and requirments

### PR DESCRIPTION
**Pull request by DarknessUnder**
**Issue**

Note which issue number(s) this Pull Request closes. All pull requests should close an issue, if one does not exists, make one. This helps us plan and discuss before we decide on implementation

*Example:*
 closes #526
 closes #527

**Database Changes/Migrations**

No change to data/API

**Test Procedure**

> 1. Create a branch
> 2. Commit a change to that branch
> 3. Push branch
> 4. Create Pull Request and fill in PR template

**Photos**
Screen Shot 2022-04-01 at 4 24 12 PM

**Additional Info**

More info that would help people review your change

### CODE REVIEW

**Review by lhain08 — APPROVED** [review]
I can't read Javascript but that all looks pretty to me.

File: src/web/src/pages/Pathway.vue [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            <div v-for="(item, itemName) in showPath" :key="itemName">
+                <h4 style="color:#3395ff">{{ itemName + ": " }}</h4>
+                <li v-for="course in item" :key="course" v-on:click="goPage(course)" class="courseInPath">{{ course }}</li>
+                <br>
+            </div>
+          </div>
+      </b-modal>
+
+      <b-row>
+        <!-- splited categories into 2 arrays, so we can have 2 columns -->
```
Comment (splanco) [inline src/web/src/pages/Pathway.vue:29]: typo in comment >:(

File: src/web/src/pages/Pathway.vue [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+                      </div>
+                  </div>
+                </div>
+
+              </b-row>
+            </b-col>
+          </b-row>
+        </b-col>
+
+        <!-- splited Pathways in alphabet order -->
```
Comment (splanco) [inline src/web/src/pages/Pathway.vue:73]: another typo >:((((((

File: src/web/src/pages/Pathway.vue [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+      ],
+      categories: json,
+      showPath: null,
+      cateShow: true,
+      alphShow : false
+    };
+    
+  },
+  computed:{
+    // splited categories into 2 arrays, one array = one column
```
Comment (splanco) [inline src/web/src/pages/Pathway.vue:160]: TEACH THIS PERSON HOW TO SPELL "SPLITTED" >>:(((((((((

File: src/web/src/pages/Pathway.vue [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+        } else {
+          col2.push(this.categories[i])
+        }
+      }
+      ret.push(col1);
+      ret.push(col2);
+      return ret;
+    },
+
+    // splited pathways to alphabet categories, then splited categories into 2 arrays, one array = one column
```
Comment (splanco) [inline src/web/src/pages/Pathway.vue:177]: AINT NO WAY

File: src/web/src/pages/Pathway.vue [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+        }
+      }
+
+      let alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"];
+      let ret = [];
+      let col1 = [];
+      let col2 = [];
+      var half_length = Math.ceil(cols.length / 2);
+      var count = 0;
+      // splited pathways to alphabet categories, then splited categories into 2 arrays
```
Comment (splanco) [inline src/web/src/pages/Pathway.vue:193]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: src/web/src/pages/Pathway.vue [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+            while (index < tmp['Pathways'].length) {
+              if (cols[m]['Name'][0] < tmp['Pathways'][index]['Name'][0]) {
+                break;
+              }
+              index++;
+            }
+            tmp['Pathways'].splice(index, 0, cols[m]);
+          }
+        }
+        // splited categories into 2 arrays
```
Comment (splanco) [inline src/web/src/pages/Pathway.vue:211]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Review by splanco — APPROVED** [review]
splited

**Pull request merged**